app/pages/home-common/featured-project.jsx

```diff
@@ -10,14 +10,16 @@ Featured Project Component
 
 import React from 'react';
 import { Link } from 'react-router';
+import Thumbnail from '../../components/thumbnail';
 
 const FeaturedProject = (() =>
   <section className="home-featured">
     <h1 className="secondary-kicker">Featured Project</h1>
     <div className="home-featured-images">
-      <img
+      <Thumbnail
         alt="Steller Sea Lion"
-        src="/assets/featured-projects/featured-project-20170801-steller.jpg"
+        src="//www.zooniverse.org/assets/featured-projects/featured-project-20170801-steller.jpg"
```

Comment:
This won't work for preview branch deploys if the assets aren't in the www.zooniverse.org bucket path due to the nginx rewrites.

Reply:
Turns out mapping staging branches to S3 bucket paths isn't simple. I'm not sure if that's something that should be handled with some extra logic in the `Thumbnail` component, or deferred to the backend somewhere.

If it's handled in the javascript, we'll need a new component method that can take a path from the image src, determine whether or not it contains a hostname, map that hostname to an S3 bucket path then return the S3 path.